src/details/MovieDetailsCard.js

```diff
@@ -1,21 +1,24 @@
 import React, { Component } from 'react';
-import { fetchMovieDetails } from '../APICalls'
 import { Link } from 'react-router-dom';
-import './MovieDetailsCard.css'
-import VideoSection from '../VideoSection/VideoSection'
+import './MovieDetailsCard.css';
+import VideoSection from '../VideoSection/VideoSection';
+import { fetchMovieDetails } from '../APICalls';
 
 class MovieDetailsCard extends Component {
-  constructor({ movieID }) {
+  constructor({ movieID, movie }) {
     super()
     this.state = {
       id: movieID,
-      movie: null,
+      movie: movie || null,
       error: null
     }
   }
 
+
   render() {
-    document.querySelector('.search-bar').classList.add('hidden')
+    if (document.querySelector('.search-bar')) {
+      document.querySelector('.search-bar').classList.add('hidden')
+    }
```

Comment:
As discussed during paired programming, we should be able to replace these querySelectors by passing rememberSearch history down to the Form as props. Will add in next PR